client/state/happychat/selectors.js

```diff
@@ -40,8 +41,12 @@ export const HAPPYCHAT_CHAT_STATUS_PENDING = 'pending';
  */
 export const getGroups = ( state, siteId ) => {
 	const groups = [];
+	const siteDetails = getSite( state, siteId );
 
-	if ( isJetpackSite( state, siteId ) ) {
+	if ( isATEnabled( siteDetails ) ) {
+		// AT sites should go to WP.com even though they are jetpack also
+		groups.push( HAPPYCHAT_GROUP_WPCOM );
```

Comment:
NM regarding both groups since I just realized it would send Jetpack users to the WPCOM support group which wouldn't be staffed with Jetpack specific operators.